integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -13480,8 +13486,8 @@ msgstr ""
 #~ msgid "Relationship"
 #~ msgstr "Verhältnis"
 
-#~ msgid "WebApp-URL"
-#~ msgstr "WebApp-URL"
+#~ msgid "Webapp-URL"
```

Comment:
Does switching from "Web**A**pp-URL" to "Web**a**pp-URL" in the msgid have any effect?